src/pages/Loans/LoansOverview/components/BorrowAssetsTable/BorrowAssetsTable.tsx

```diff
@@ -40,7 +40,9 @@ type BorrowAssetsTableProps = {
 
 const BorrowAssetsTable = ({ assets, onRowAction, disabledKeys }: BorrowAssetsTableProps): JSX.Element => {
   const { t } = useTranslation();
+  // ray test touch <
   const prices = useGetPrices();
```

Comment:
I think we should refactor `useGetPrices` use cases within the lending and borrowing scope.